Automattic/wp-calypso
# Update signup goals page skip to dashboard set site_intent to build

#### Testing/Review instructions

Test: short
Review: short

#### Proposed Changes

Currently, users that click the `Skip to dashboard` link on the top right-hand corner of the Goals onboarding step will be navigated to the home screen with the following site options:

`site_intent` = ""
`launchpad_screen` = false

The changes in this PR will set the `site_intent` site option to `build` if the user clicks `Skip to dashboard` on the Goals page. This will also enable the launchpad and set the `launchpad_screen` site option to `full` before the user is navigated to the home screen.

P2 post to give heads up: paYE8P-2yP-p2

Goals step:
<img width="1508" alt="image" src="https://user-images.githubusercontent.com/10482592/216123805-a7a15a5a-2a92-4d50-8750-373a4516f835.png">


#### Testing Instructions



* Checkout this branch
* Navigate to the onboarding flow (Click `< Switch Site` on the Calypso sidebar and then click `Add a new site`)
* Create your free site and you should land on the Goals step.
* Do not select a goal and click `Skip to dashboard`. The `site_intent` and `launchpad_screen` values should be updated. The home screen should redirect you to the launchpad screen (There may be a bug that requires multiple refreshes before you redirected - )
* Create a new site and test another goal such as `Write & Publish` to verify the correct `site_intent` value is set

#### Pre-merge Checklist





Related to #72870
